itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/launch/configuration/AbstractLaunchConfigurationTest.java

```diff
@@ -66,7 +60,7 @@ public void createNewQuarkusConfiguration(String projectName) {
 		new PushButton("OK").click();
 
 		new PushButton("Apply").click();
```

Comment:
For Apply there is a function: https://github.com/eclipse/reddeer/blob/37f27affd709a9622d84e8d29d047dc78a0de1b4/plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/debug/ui/launchConfigurations/LaunchConfiguration.java#L48

Reply:
fixed.